Training/python/config_parse.py

```diff
@@ -177,9 +177,16 @@ def create_scaling(content_scaling: dict, content_cfg: dict) -> str:
                 string += "std::vector<std::vector<float>> "
                 string += subg + " = "
                 var_string = []
-                for var_i, (var, var_params) in enumerate(content_scaling[FeatureT].items()):
-                    assert var in content_cfg['Features_all'][FeatureT][var_i].keys() # check if there is such feature in training cfg
-                    if var in content_cfg['Features_disable'][FeatureT]: continue
+
```

Comment:
For the sake of consistency I would also change the iteration over the feature groups based on the training config (instead of the scaling cfg currently):
https://github.com/cms-tau-pog/TauMLTools/blob/f6342fe68bc080731bcf6f84193cdd19011770cc/Training/python/config_parse.py#L172